### PR TITLE
Docs formatting

### PR DESCRIPTION
**Pull request by navidcy**

### CODE REVIEW

**Comment by jishnub** [comment]
Could you resolve the merge conflicts? Otherwise this looks like an improvement

File: doc/src/manual/mathematical-operations.md [hunk 1]
```diff
@@ -457,7 +457,7 @@ Juxtaposition parses like a unary operator, which has the same natural asymmetry
 Julia supports three forms of numerical conversion, which differ in their handling of inexact
 conversions.
 
-  * The notation `T(x)` or `convert(T,x)` converts `x` to a value of type `T`.
+  * The notation `T(x)` or `convert(T, x)` converts `x` to a value of type `T`.
```
Comment (jishnub) [inline doc/src/manual/mathematical-operations.md:460]: This text isn't strictly true for a constructor. Perhaps only `convert` should be mentioned here?

File: doc/src/manual/mathematical-operations.md [hunk 2]
```diff
@@ -557,13 +557,13 @@ See [Conversion and Promotion](@ref conversion-and-promotion) for how to define
 | [`cbrt(x)`](@ref), `∛x`  | cube root of `x`                                                           |
 | [`hypot(x, y)`](@ref)    | hypotenuse of right-angled triangle with other sides of length `x` and `y` |
 | [`exp(x)`](@ref)         | natural exponential function at `x`                                        |
-| [`expm1(x)`](@ref)       | accurate `exp(x)-1` for `x` near zero                                      |
-| [`ldexp(x, n)`](@ref)    | `x*2^n` computed efficiently for integer values of `n`                     |
+| [`expm1(x)`](@ref)       | accurate `exp(x) - 1` for `x` near zero                                 λ   |
```
Comment (jishnub) [inline doc/src/manual/mathematical-operations.md:560]: Is the trailing `λ` accidental?

Comment (navidcy) [reply doc/src/manual/mathematical-operations.md:560]: Yes!!

File: doc/src/manual/mathematical-operations.md (hunk 1 above)
Comment (navidcy) [reply doc/src/manual/mathematical-operations.md:460]: I don't quite understand what you mean here.

**Review by jishnub — APPROVED** [review]

**Pull request merged**